### PR TITLE
Matplotlib example

### PR DESCRIPTION
**Pull request by baskrahmer**
## What does this PR do?

Example of using Matplotlib animations in Sphinx-Gallery. Basically the only prerequisite is that `matplotlib_animations` is set to `True` in `conf.py`, which seems to already be the case :)

@borda

<details>
  <summary>Before submitting</summary>

- [ ] Was this **discussed/agreed** via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/Lightning-AI/torchmetrics/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to **update the docs**?
- [ ] Did you write any new **necessary tests**?

</details>

<details>
  <summary>PR review</summary>

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

</details>

## Did you have fun?

Make sure you had fun coding 🙃

----
📚 Documentation preview 📚: https://torchmetrics--2536.org.readthedocs.build/en/2536/

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.PHONY: clean test get-sphinx-template docs env data
+.PHONY: clean test get-sphinx-template docs live-docs env data
```
Comment (Borda) [inline Makefile:1]: lets keep these change in #2518

**Comment by SkafteNicki** [comment]
@baskrahmer what does the code have to do with torchmetrics? or is it just dummy code for now?

**Comment by baskrahmer** [comment]
> @baskrahmer what does the code have to do with torchmetrics? or is it just dummy code for now?

Hey, it's just dummy code. Since we have some Matplotlib animations implemented in the documentation Gallery by now, this PR can be closed 😄

**Comment by SkafteNicki** [comment]
> > @baskrahmer what does the code have to do with torchmetrics? or is it just dummy code for now?
> 
> Hey, it's just dummy code. Since we have some Matplotlib animations implemented in the documentation Gallery by now, this PR can be closed 😄

Alright, fine by me 😄 but feel free to reopen if necessary

**Pull request closed without merging**